src/Octoshift/Handlers/WaitForMigrationCommandHandler.cs

```diff
@@ -108,20 +108,22 @@ private async Task WaitForRepositoryMigration(string migrationId, string githubP
             if (RepositoryMigrationStatus.IsSucceeded(state))
             {
                 _log.LogSuccess($"Migration {migrationId} succeeded for {repositoryName}");
+                _log.LogInformation($"Migration log available at {url} or by running `gh gei download-logs`");
                 return;
             }
 
             if (RepositoryMigrationStatus.IsFailed(state))
             {
                 _log.LogError($"Migration {migrationId} failed for {repositoryName}");
+                _log.LogInformation($"Migration log available at {url} or by running `gh gei download-logs`");
                 throw new OctoshiftCliException(failureReason);
             }
 
             _log.LogInformation($"Migration {migrationId} for {repositoryName} is {state}");
             _log.LogInformation($"Waiting {WaitIntervalInSeconds} seconds...");
             await Task.Delay(WaitIntervalInSeconds * 1000);
 
-            (state, repositoryName, failureReason) = await githubApi.GetMigration(migrationId);
+            (state, repositoryName, failureReason, url) = await githubApi.GetMigration(migrationId);
```

Comment:
```suggestion
            (state, repositoryName, failureReason, migrationLogUrl) = await githubApi.GetMigration(migrationId);
```

Reply:
Changed in 486316ab29031cdf5ea320131592eadb300a2a3c :tada: